src/map/packet_system.cpp

```diff
@@ -7978,22 +7996,11 @@ void SmallPacket0x106(map_session_data_t* const PSession, CCharEntity* const PCh
 void SmallPacket0x109(map_session_data_t* const PSession, CCharEntity* const PChar, CBasicPacket& data)
 {
     TracyZoneScoped;
-    CItemContainer* PStorage = PChar->getStorage(LOC_INVENTORY);
-    if (PStorage == nullptr)
-    {
-        return;
-    }
 
-    for (uint8 slotID = 1; slotID <= PStorage->GetSize(); ++slotID)
+    if (PChar->isSettingBazaarPrices)
```

Comment:
It seems like there are const methods to expose the flags for all of these new bools, can we do the same here please? Maybe making the internal flags private?

Reply:
Those const methods were required to access single bits of a struct iirc
here we would get some absolutely insane naming like

`setIsSettingBazaarPrices(true)`
and `getIsSettingBazaarPrices()`
Not sure if we want that, there really isn't a better name for that variable because that's whats happening. The bazaar flag needs to away while they are _setting_ prices, because they still technically have items in their bazaar in the db because the prices haven't updated yet

Reply:
Makes sense, carry on without changes 👍